poster/styles/styles.scss

```diff
@@ -11,480 +7,451 @@ sans-serif;
     }
 
-    .body-linefifty {
-        margin-left: 15vw;
+    &fifth {
+        font-family: $thick-display;
+        font-size: 28vw;
+        grid-column: 5/6;
+        grid-row: 1/7;
+        margin-left: -6vw;
+        transform: rotate(-30deg);
     }
 
-    .body-linefiftyone {
-        margin-left: 8vw;
+    &sixth {
+        font-family: $thin-display;
+        grid-column: 6/7;
+        grid-row: 14/20;
     }
 
-    .body-linefiftytwo {
-        margin-left: 15vw;
+    &seventh {
+        font-family: $thin-display;
+        grid-column: 7/8;
+        grid-row: 10/16;
     }
+}
 
-    .body-linefiftythree {
-        display: block;
-        font-size: 1.2vw;
-        margin-left: 60vw;
-        margin-top: -7vw;
+@media (max-width:850px) {
+
+    /*redefined margins for body*/
+    .line__ {
+        &one {
+            padding-top: 1vw;
+        }
+
+        &four {
+            margin-left: 49vw;
+        }
+
+        &five {
+            margin-top: -3vw;
+            margin-left: 39vw;
+        }
+
+        &twelve {
+            margin-bottom: -3vw;
+        }
+
+        &twentytwo {
+            margin-left: 29vw;
+            margin-bottom: 4vw;
+        }
+
+        &twentyfive {
+            margin-left: 6vw;
+        }
+
+        &thirty {
+            margin-bottom: -3vw;
+        }
+
+        &thirtyone {
+            margin-left: 6vw;
+        }
+
+        &thirtytwo {
+            margin-left: 2vw;
+        }
+
+        &thirtytwosecond {
+            margin-left: 0vw;
+        }
+
+        &fortythree {
+            margin-top: -7.5vw;
+            margin-left: 50vw;
+            margin-bottom: -4vw;
+        }
+
+        &fortyfour {
+            margin-left: 47vw;
+        }
+
+        &fortyfive {
+            margin-left: 43vw;
+        }
+
+        &fortysix {
+            margin-left: 38vw;
+        }
+
+        &fortyseven {
+            margin-left: 33vw;
+        }
+
+        &fortyeight {
+            margin-left: 26vw;
+        }
+
+        &fortynine {
+            margin-left: 22vw;
+        }
+
+        &fifty {
+            margin-left: 15vw;
+        }
+
+        &fiftyone {
+            margin-left: 8vw;
+        }
+
+        &fiftytwo {
+            margin-left: 15vw;
+        }
+
+        &fiftythree {
+            display: block;
+            font-size: 1.2vw;
+            margin-left: 60vw;
+            margin-top: -7vw;
+        }
     }
 
     /*bottom half grid establishes and styles 1 column for better readablity*/
     .bottomhalf {
         display: grid;
         grid-template-columns: 1fr;
         grid-template-rows: 50% 50%;
-        font-size: $font-sizemediaqone;
         line-height: 4vw;
         margin-left: 6vw;
-    }
 
-    .firstcoloumn {
-        grid-column: 1/2;
-        grid-row: 1/2;
-    }
+        h2 {
+            font-size: 5vw;
+            font-weight: bold;
+        }
 
-    h2 {
-        font-size: 5vw;
-        font-weight: bold;
-    }
+        div,
```

Comment:
I changed the `span`s to `div`s in your markup -- since `div`s are block-level elements, this meant I could also eliminate the `br` tags 🙌 
This change required me to add the `div` to this set of selectors.